File: src/main/java/com/sica/backend/dto/AttendanceResponse.java
package com.sica.backend.dto;

import java.time.LocalDate;
import java.time.LocalDateTime;

public record AttendanceResponse(
    String id,
    String estudianteId,
    String estudianteNombre,
    String estudianteFoto,
    String profesorId,
    String profesorNombre,
    String materiaId,
    String materiaNombre,
    LocalDate fecha,
    String estado,
    Boolean justificado,
    LocalDateTime creadoEn,
    LocalDateTime actualizadoEn
) {}

File: src/main/kotlin/com/sica/backend/dto/AttendanceResponse.kt
package com.sica.backend.dto

import java.time.LocalDate
import java.time.LocalDateTime

data class AttendanceResponse(
    val id: String,
    val estudianteId: String? = null,
    val estudianteNombre: String? = null,
    val estudianteFoto: String? = null,
    val profesorId: String? = null,
    val profesorNombre: String? = null,
    val materiaId: String,
    val materiaNombre: String,
    val fecha: LocalDate,
    val estado: String,
    val justificado: Boolean,
    val creadoEn: LocalDateTime? = null,
    val actualizadoEn: LocalDateTime? = null
)
